src/index.js

```diff
@@ -81,32 +81,32 @@ const perspective = Math.floor(FXRandomBetween(0.01, 0.08) * 100) / 100
 const missingTiles = Math.floor(FXRandomBetween(0.3, 0.8) * 10) / 10
 
 const stylesClasses = [
-  AdaAdaAdaStyle,
-  RVigStyle,
-  ElsifStyle,
-  EstienneStyle,
-  FrederativeStyle,
-  DavidEsqStyle,
-  ShuhblamStyle,
-  GorikStyle,
-  AnaglyphicStyle,
-  PhilosophieStyle,
-  WilkeStyle,
-  RobinMetcalfeStyle,
-  CamilleRoux2Style,
-  CamilleRouxStyle,
-  BoilerplateStyle,
-  DemoStyle,
-  AzeemStyle,
-  Makio64Style,
-  MandyBrigwellStyle,
-  BridgeTunnelStyle,
-  DreyStyle,
-  LunareanStyle,
-  GrosggStyle,
-  bfosStyle,
-  DevnullStyle,
-  Nobody,
+  // AdaAdaAdaStyle,
```

Comment:
You can't commit these changes

Reply:
Ooops, I just reverted the change on index :)

thanks!